app/graphql/resolvers/concerns/collection.rb

```diff
@@ -93,6 +95,16 @@ def authorized_scope
       def permit_tags?(tags)
         TagFiltering.tags_supported?(model_class) && tags.present?
       end
+
+      # Store the scope without applying any filtering by tags on it. This is
+      # required for listing the available tags in a view without any influence
+      # of the tagging scope. The selection is based on the second argument of
+      # the lambdas defined by each filter.
+      def store_wide_scope!(filter = ->(_, nil: nil) {})
+        return yield if %i[_tags_filter _pagination_filter].include?(filter.parameters[1][1])
+
+        context[:wide_scope] = yield
```

Comment:
Looks like the filters are not stored, so why not just store the `base_scope`? What am I missing?

This implementation seems very rigid. Can you think of a simplified one that can be better understandable?